docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,809 @@
+{{< /alert >}}
+
+### Configure your first machine
+
+#### Configure the sensor
+
+1. Navigate to the **CONFIGURE** tab of the machine details page in the [Viam app](https://app.viam.com) for your first machine.
+2. Click the **+** (Create) button and click **Component** from the dropdown.
+   Click **sensor**, then search for `sds011` and click **sds001:v1** from the results.
+3. Click **Add module**.
+   This adds the {{< glossary_tooltip term_id="module" text="module" >}} that provides the sensor model that supports the specific hardware we are using for this tutorial.
+
+   {{<imgproc src="/tutorials/air-quality-fleet/add-sensor-module.png" resize="x1100" declaredimensions=true alt="The Add Module button that appears after you click the model name." style="max-width:600px" >}}
+
+4. Give the sensor a name like `PM_sensor` and click **Create**.
+5. In the newly created **PM_sensor** card, replace the contents of the attributes box (the empty curly braces `{}`) with the following:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "usb_interface": "<REPLACE WITH THE PATH YOU IDENTIFY>"
+   }
+   ```
+
+6. <a name="usb-path"></a>Now you need to figure out which port your sensor is connected to on your board.
+   SSH to your board and run the following command:
+
+   ```sh{class="command-line" data-prompt="$"}
+   ls /dev/serial/by-id
+   ```
+
+   This should output a list of one or more USB devices attached to your board, for example `usb-1a86_USB_Serial-if00-port0`.
+   If the air quality sensor is the only device plugged into your board, you can be confident that the only device listed is the correct one.
+   If you have multiple devices plugged into different USB ports, you may need to choose one path and test it, or unplug something, to figure out which path to use.
+
+   Now that you have found the identifier, put the full path to the device into your config, for example:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "usb_interface": "/dev/serial/by-id/usb-1a86_USB_Serial-if00-port0"
+   }
+   ```
+
+7. Save the config.
+   Your machine config should now resemble the following:
+
+   {{<imgproc src="/tutorials/air-quality-fleet/configured-sensor.png" resize="x1100" declaredimensions=true alt="Configure tab showing PM sensor and the sensor module configured." style="max-width:600px" >}}
+
+#### Configure data capture and sync
+
+You have configured the sensor so the board can communicate with it, but sensor data is not yet being saved anywhere.
+Enable [data capture](/data/capture/) and [cloud sync](/data/cloud-sync/) so that data from your air quality sensor will be first stored on the machine and then pushed up to the cloud where you can access all the data from your sensors remotely.
+You'll be able to access data from all sensors in all locations, and when you're ready, you can give customers [access](/fleet/rbac/) to the data from the sensors in their locations.
```

Comment:
```suggestion
You have now configured the sensor so the board can communicate with it.
You could at this point build a dashboard that shows the sensor's current readings.
To allow you to show past data and make such a dashboard more useful, you need to store the data from your sensor. 
In the Viam Platform you can use the data management service to capture and store your sensor data and sync it with the cloud.
Once you configure more than one sensor, you'll be able to access data from all sensors in all locations, and when you're ready, you can give customers [access](/fleet/rbac/) to the data from the sensors in their locations.
```
This is more what I'm thinking. Adding why you'd want to capture this data, I.e. To be able to show historic data. 

This is example wording, feel free to change

Reply:
I'd rather not talk about a dashboard they could build that is different from the one they are going to build, especially since the dashboard they are going to build here doesn't really display historical data as such (though it technically uses some historical data for averages and for sensors that aren't currently live).

The main reason the data service is useful here is that it lets them tag data and then get data filtering by tag, location, and machine, so the same dashboard code scales with their fleet and works for any location versus needing to code in getting a robot client and `get sensor FromRobot` for every machine. I feel that the last sentence says this. Maybe this does a better job of emphasizing it?
```suggestion
You have configured the sensor so the board can communicate with it, but sensor data is not yet being saved anywhere.
In this section you'll configure [data capture](/data/capture/) and [cloud sync](/data/cloud-sync/) so that data from your air quality sensor will be stored on the machine and then synced to the cloud where you can access it remotely.
Once you configure all your sensing machines, you'll be able to access data from all sensors in all locations from one central hub, and when you're ready, you can give customers [access](/fleet/rbac/) to the data from the sensors in their locations.
```

Reply:
I think there is a logical issue here. If the business doesn't care about historical data there is no real reason to use data management. 
You pay for data upload, management, and egress... So a business would only do that if they care about keeping the data. 

The code here could be changed fairly simply to 
Connect with your api key, and I think you could even then list all the organisations and locations you have access to and thus get all the machines and for each get the sensor and the current reading. You could even call this every few seconds and then average over the past 5. So why would you not do that? What's bad about creating many robot clients to access the data?

I realize offline sensors would then cause an issue but I probably also don't want to open a dashboard and see live-looking data that's from two weeks ago.

You don't need to change the dashboard. Just pointing out that they can see and access historical data and add that to their dashboard is fine.

You touch on it at the very end as you say. I want that to be the start of the section. I think "why would you add data management" needs to be clear right away, ideally in the first one or two sentences rather than at the very end.

Reply:
Mm I see--you're right about the cost. This is interesting.

RE the bad thing about creating robot clients: You could use the Python SDK to establish an appclient with access to all machines in all locations in the org, and use that to get the ids of all machines in each location, and then establish robotclients (with the typescript SDK?) for each machine, but you wouldn't be able to loop through just the air sensor machines because you wouldn't know which machines were air sensor machines, unless your entire org had no other machines in it, right? And, if any of those machines collected non-air-quality data, that data would also be included right? Maybe you could name all air sensor machines "air-sensor-bend1," "air-sensor-bend2," "air-sensor-nyc1," and similarly have a consistent naming scheme for the air quality sensors so you'd know which sensor to get readings from, but it'd be hacky. So the nice thing about using the data service is that you can efficiently tag all the data and get exactly the data you want, with more flexibility.
Is there another way to do this without using data management that I haven't thought of? Also wouldn't this currently require using the Python SDK in addition to the TS SDK because appclient is currently supported only in Python?

Reply:
Is the question here whether this tutorial gains anything from historical data/data capturing?

Reply:
The question is would someone realistically choose to use the data management service and capture and sync data with it just so they can easily access all machines with one api call if the data they are accessing is current data. I think the answer to that is no.

I think people would only use the data management service if they needed the historical data. For something. And I think we need to start this section by providing the reason why you'd add the data management service (which costs money).

If its easy then yes please update the dashboard to show the last hour, day, 7 days or something like that. That would be best. If that's not easy I think it's ok to just say you want that data for future analysis and when you build the dashboard say "because we have access to all historical data from the sensor, we could also..."

Reply:
There are lots of tools that aren't grafana that they could use. I think it's not important how they access and visualise the data on the dashboard. It's more that we need to tell the story that they want to keep this data for something and that's why they use the data management service. 

I think adding grafana to this tutorial may be a stretch but you could add a link to it in the next steps section saying that you could do nice visualisations with grafana as well since you have all the data

Reply:
Essentially the 
"Current pollution data is great but you and your customers may also want to keep track of data to be able to look at it over time." and then say that you can do that with the data management service.

Reply:
It would be non-trivial for me to make the code work that way, especially without it loading very slowly, and though I could put in hard-coded HTML to take new screenshots, it'd take a while to actually test the code properly since I don't currently have a week of multi-machine data, so I'd rather merge with the dashboard as-is. Should I make a ticket to change the code in the future or is that overkill?

I do have a note about Grafana after the data test section, and I also have a Grafana card in the next steps section. Actually right now I'm adding a provisioning card to next steps too, but yes Grafana is mentioned.

```suggestion
You have configured the sensor so the board can communicate with it, but sensor data is not yet being saved anywhere.
Enable [data capture](/data/capture/) and [cloud sync](/data/cloud-sync/) so that data from your air quality sensor will first be stored on the machine and then synced to the cloud where you can access historical sensor data and see trends over time.
Once you configure the rest of your sensing machines, you'll be able to remotely access data from all sensors in all locations, and when you're ready, you can give customers [access](/fleet/rbac/) to the data from the sensors in their locations.
```

Reply:
Then let's keep as is with the copy change you just suggested. Thanks!